langgenius/dify
# Add Lindorm as a VDB choice

# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.


# Description
Lindorm is a multi-model nosql-database, providing the capability of full-text search, vector search and hybrid search

This is the issue link
Fixes #10064 [documentation](https://github.com/langgenius/dify/issues/10064#issue-2623364483)

## Type of Change


# Testing Instructions

To run the test in dify/api/tests/integration_tests/vdb/lindorm, you should fill the connect information like other commercial vector-database.
